src/bp/ui-studio/src/web/views/FlowBuilder/common/EditableInput.jsx

```diff
@@ -29,6 +29,11 @@ export default class EditableInput extends Component {
   }
 
   onKeyDown = event => {
+    // Cmd or ctrl + A to select all text
+    if ((event.ctrlKey || event.metaKey) && event.keyCode === 65) {
```

Comment:
const KEY_A = 65
event.keyCode === KEY_A (and remove the comment)

Preferrably it would use the existing keyHandlers (React hotkeys), but since it's only to support a "builtin" shortcut, that's ok